Python/samples/cartpole/multi-concept.ink

```diff
@@ -0,0 +1,147 @@
+# https://docs.bons.ai/references/inkling2-reference.html
```

Comment:
Would an overall comment about the purpose of this example and the principle that it is teaching be helpful? If someone is new to this, should they be looking at machine_teacher.ink or multi-concept.ink?

Reply:
Maybe summarize the roles of ComputeDelta, GoLeft, GoRight, and PickOne?

Reply:
Good idea. Added!